io/src/csv/read_measurements.cpp

```diff
@@ -44,7 +44,7 @@ void read_measurements(measurement_collection_types::host& measurements,
 
         // Construct the measurement object.
         traccc::measurement meas;
-        std::array<typename transform3::size_type, 2u> indices{0u, 0u};
+        std::array<traccc::size_type, 2u> indices{0u, 0u};
```

Comment:
I feel quite strongly about this... Let's not introduce a new global type, just to read and write measurements. We can just use a local type inside this code.

After all, we are trying to get rid of global floating point types to begin with. :thinking:

Reply:
I don't have a strong opinion on this, I can get the size type locally from the global default algebra. That is essentially also what has been done in the code so far. The size type is still defined globally for the algebra plugin, just not as visible.